framework/render/exceptions.py

```diff
@@ -20,10 +23,19 @@ def __init__(self, http_status_code, **additional):
         super(Exception, self).__init__(msg)
```

Comment:
This should be `super(RenderFailureException, self).__init__(msg)`  or `Exception.__init__(self, msg)`.